src/models/minecraft_protocol.rs

```diff
@@ -305,6 +325,37 @@ fn read_chunk_section<S: Read>(stream: &mut S) -> ChunkSection {
     }
 }
 
+fn write_position<S: Write>(stream: &mut S, v: BlockPosition) {
+    println!(
```

Comment:
Let's not merge any println- if this is something that actually should be logged use `trace!`